cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -61,4 +66,26 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+      - name: healthz
+        image: gcr.io/google_containers/healthz-server:1.0
+        resources:
```

Comment:
10m should be fine, you have 100m now.

"outer" means a limit for the pod, such that something with 0 CPU could use whatever CPU was free in the pod